pkg/sloop/webserver/webfiles/debug.html

```diff
@@ -9,11 +9,12 @@
     <script type="text/javascript">
         document.write("<base href='/" + window.location.pathname.split('/')[1] + "/' />");
     </script>
+    <script src="webfiles/debug.js"></script>
```

Comment:
Did you do anything special for fixing the relative path issue for this file ?.

Reply:
I set the base href before making the call to reference the external js file.